src/redux/types/index.js

```diff
@@ -61,16 +61,16 @@ const messageTypes = {
  *
  * @type {{PLATFORM_USER_AUTH: string, SET_PLATFORM_EXPORT_STATUS: string, PLATFORM_GLOBAL_FILTER_HIDE: string,
  *     PLATFORM_CLEAR_NOTIFICATIONS: string, PLATFORM_ADD_NOTIFICATION: string, PLATFORM_REMOVE_NOTIFICATION: string,
- *     GET_PLATFORM_EXPORT: string}}
+ *     SET_PLATFORM_EXPORT_CREATE: string}}
  */
 const platformTypes = {
   PLATFORM_ADD_NOTIFICATION: ADD_NOTIFICATION,
   PLATFORM_REMOVE_NOTIFICATION: REMOVE_NOTIFICATION,
   PLATFORM_CLEAR_NOTIFICATIONS: CLEAR_NOTIFICATIONS,
   PLATFORM_GLOBAL_FILTER_HIDE: 'PLATFORM_GLOBAL_FILTER_HIDE',
   PLATFORM_USER_AUTH: 'PLATFORM_USER_AUTH',
-  GET_PLATFORM_EXPORT: 'GET_PLATFORM_EXPORT',
-  SET_PLATFORM_EXPORT_STATUS: 'SET_PLATFORM_EXPORT_STATUS'
+  SET_PLATFORM_EXPORT_STATUS: 'SET_PLATFORM_EXPORT_STATUS',
+  SET_PLATFORM_EXPORT_CREATE: 'SET_PLATFORM_EXPORT_CREATE'
```

Comment:
required: confirm all redux action types are still used

Reply:
they're all used, looks like we may be able to add one for existing exports. reviewing